fastlane/Appfile

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+json_key_file('') # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
+package_name('com.your.package.name') # e.g. com.krausefx.app
```

Comment:
I would expect this is our `template app id`, that makes easy for us on finding and replacing later when cloning to the real project? 🤔